ingestion/src/metadata/utils/secrets/client/loader.py

```diff
@@ -37,7 +40,9 @@ def _(_: SecretsManagerProvider) -> None:
 
 
 @secrets_manager_client_loader.add(SecretsManagerClientLoader.airflow.value)
-def _(provider: SecretsManagerProvider) -> Optional[AWSCredentials]:
+def _(
+    provider: SecretsManagerProvider,
+) -> Optional[Union[AWSCredentials, AzureCredentials]]:
```

Comment:
nit, I think we can define the type as `ExternalSecretsManager`

Reply:
good point, thank you